### DISCUSSION TITLE
Article Preview

### DISCUSSION THREAD

**Discussion by AndyM48** [martinrotter/rssguard#1396] [Ideas]
It would be a nice feature to be able to choose from alternative article preview layouts.

**Comment by martinrotter** [comment]
In RSS Guard there is a concept of skins.

https://rssguard.readthedocs.io/en/stable/features/skins.html

Layout of article preview is part of skin feature and consists of number of HTML/CSS templates which are assembled in runtime together to show an article.

Currently there is only one (dark, light) official skin but as per documentation some users actually created their own skins. I currently do not plan to create another official skin with alternative article preview layout. Maybe someone of users will see this and provide theirs.

**Comment by AndyM48** [comment]
Anyone?

Feel free to post your Article Preview alternate skin here.
